OsmAnd/src/net/osmand/plus/track/fragments/ReadDescriptionFragment.java

```diff
@@ -123,29 +116,6 @@ private void setupToolbar(@NonNull View view) {
 		}
 	}
 
-	private void setupImage(@NonNull View view) {
-		String imageUrl = getImageUrl();
-		if (imageUrl == null) return;
-
-		AppCompatImageView ivImage = view.findViewById(R.id.main_image);
-		PicassoUtils picasso = PicassoUtils.getPicasso(app);
-		RequestCreator rc = Picasso.get().load(imageUrl);
-		WikivoyageUtils.setupNetworkPolicy(app.getSettings(), rc);
```

Comment:
We don't need it anymore?

Reply:
It is got back and tested for on/off, including cleared images cache.

Reply:
38cd377ffb